contrib/utilities/indent

```diff
@@ -44,3 +44,23 @@ fi
 # with up to 10 in parallel
 find tests include source examples \( -name '*.cc' -o -name '*.h' \) -print | xargs -n 50 -P 10 astyle --options=contrib/utilities/astyle.rc
 
+# format .inst.in files. We need to replace \{ and \} because it confuses
+# astyle.
+format_inst()
+{
+    f=$1
+    cp $f $f.tmp
+    sed -i 's#\\{#{ //#g' $f.tmp
+    sed -i 's#\\}#} //#g' $f.tmp
+    astyle --quiet $f.tmp
```

Comment:
I tried this at first, but one of our settings breaks ``{ //`` into separate lines, so my hack above won't work. I was too lazy to isolate that setting, write a new astyle config, etc..